src/ui/digital-exchange/components/common/ComponentListViewModeSwitcherContainer.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setDEComponentsListViewMode } from 'state/digital-exchange/components/actions';
+import { getDEComponentListViewMode } from 'state/digital-exchange/components/selectors';
+import PropTypes from 'prop-types';
+
+
+const ComponentListViewModeSwitcher = ({ viewMode, changeViewMode }) => {
+  function switchToGridView(e) {
```

Comment:
I don't see use of `function` in our codebase. @nicpuddu do we have any guideline about this?
Generally speaking I like functions, but in this case I'd like to keep the codebase consistent and use an arrow function with `const` or a class method

Reply:
I agree, either use an anonymous arrow function or declare the function as a constant.